deploy/layered-sre-authorization/02-layered-cs-sre-admin.SubjectPermission.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: managed.openshift.io/v1alpha1
+kind: SubjectPermission
+metadata:
+  name: layered-cs-sre-admin
+  namespace: openshift-rbac-permissions
+spec:
+  clusterPermissions:
+  - dedicated-admins-cluster
```

```suggestion
  - layered-cs-sre-admin
```

This should not be the `dedicated-admins-cluster` ClusterRole

---

I'm trying to achieve giving the group layered-cs-sre-admin the dedicated-admins-cluster role available clusterwide, and the layered-cs-sre-admin role available in all the non redhat namespaces. Am I applying this wrong?

---

You could add layered-cs-sre-admin to this, it's an array.  Why does CS SRE need dedicated-admin permissions in cluster?  I expect your `layered-cs-sre-admin` ClusterRole to have everything CS SRE needs in a customer namespace and that those permissions are something less than dedicated-admin.

---

Ok, Interesting. My view is that we have the same as the customer and a subset more. This is purely from the nature of the type of work we do. We are managing middleware products with customers more than on behalf. An example here is products dynamically creating namespaces that hold config. Customers can manage this but so can CS SRE.

---

Ok, I see my disconnect.  You're ClusterRole reads to me like a project level permissions.  I missed the clusterPermissions will create the ClusterRoleBinding.  Your layered-cs-sre-admin ClusterRole is too broad to grant as a ClusterRoleBinding, please split it into two ClusterRoles, one for cluster scope resources (project, namespace, user, etc) and project resources (everything else).  In the SubjectPermissions the cluster scope ClusterRole goes into `clusterPermissions`, the project scope ClusterRole goes into `permissions`.

---

I also see what you were originally asking.  I've splt the cluster scope and project resources out (layered-cs-sre-admin-cluster and layered-cs-sre-admin-project)